### PR TITLE
Update the Changelog and Readme with the most recent changes.

### PR DESCRIPTION
**Pull request by vRallev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
 
 ## Next Version
 
+* Change the `replaces` attribute from a single class to an array. This gives the API more flexibility and avoids redundant classes. E.g. one Dagger module with several binding and provider methods may wish to replace multiple other Dagger modules:
```
Comment (vRallev) [inline CHANGELOG.md:5]: Remove the `:` at the end.

File: README.md [hunk 2]
```diff
@@ -93,6 +93,31 @@ the Dagger component, e.g.
 interface AppComponent
 ```
 
+## Contributed bindings
+
+The `@ContributesBinding` annotation generate a Dagger binding method for an annotated class and
```
Comment (FrostRocket) [inline README.md:98]: generate -> generates

File: README.md [hunk 3]
```diff
@@ -93,6 +93,31 @@ the Dagger component, e.g.
...
+
+class RealAuthenticator @Inject constructor() : Authenticator
+
+@Module
+@ContributesTo(AppScope::class)
+abstract class AuthenticatorModule {
+  @Binds abstract fun bindRealAuthenticator(authenticator: RealAuthenticator): Authenticator
+}
+```
+This is a lot of boilerplate if you always want to use `RealAuthenticator` when injecting
```
Comment (FrostRocket) [inline README.md:111]: Nice 💯

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,23 @@
...
+class RealAuthenticator @Inject constructor() : Authenticator
+
+// The generated and automatically included Dagger module would look similar like this:
+@Module
+@ContributesTo(AppScope::class)
+abstract class AuthenticatorModule {
+  @Binds abstract fun bindRealAuthenticator(authenticator: RealAuthenticator): Authenticator
+}
+```
+* Support nested classes for contributed Dagger modules and component interfaces if the outer class uses a different scope #45.
```
Comment (zach-klippenstein) [inline CHANGELOG.md:20]: Did you want to link to PRs for the other items in the list as well?

File: README.md (hunk 3 above)
Comment (zach-klippenstein) [reply README.md:111]: Holy shit. Have you tweeted this snippet out? This is the code I've always wanted to write. It's so beautiful…

File: CHANGELOG.md (hunk 4 above)
Comment (vRallev) [reply CHANGELOG.md:20]: There were no external issues for them. I only link to issues and no PRs usually.

File: README.md (hunk 3 above)
Comment (vRallev) [reply README.md:111]: Will tweet later today when it's out :)

**Comment by vRallev** [comment]
I'm going to merge and release the new version tomorrow. It's too late for today.

**Pull request merged**